.gitignore

```diff
@@ -2,6 +2,7 @@ build
 aqt/version.py
 dist
 venv
+build_venv
```

Comment:
```suggestion

```

Hi. I found that virtualenv will generate a ".gitignore" file in the virtual environment directory automatically, so it's unnecessary to ignore it manually.

> ## [v20.0.21 (2020-05-20)](https://virtualenv.pypa.io/en/latest/changelog.html#v20-0-21-2020-05-20)
> ### [Features - 20.0.21](https://virtualenv.pypa.io/en/latest/changelog.html#features-20-0-21)
> - Generate ignore file for version control systems to avoid tracking virtual environments by default. Users should remove these files if still want to track. For now we support only **git** by **gaborbernat**. ([#1806](https://github.com/pypa/virtualenv/issues/1806))